### PR TITLE
Improve documentation regarding delayed startup

### PR DESCRIPTION
**Pull request by WhyNotHugo**
Slightly improve the documentation and mention dbus-activation.

This is a feature that's already implemented, so it's definitely worth mentioning it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,23 @@ A lightweight notification daemon for Wayland. Works on Sway.
 
 mako implements the [GNOME Desktop Notifications Specification][gnome-draft].
 
-Feel free to join the IRC channel: ##emersion on irc.freenode.net.
+Feel free to join the IRC channel: #emersion on irc.freenode.net.
```
Comment (emersion) [inline README.md]: The channel name is ##emersion, not #emersion.

Comment (WhyNotHugo) [reply README.md]: My bad, though it was a typo :sweat:

File: README.md [hunk 2]
```diff
@@ -12,11 +12,19 @@ Feel free to join the IRC channel: ##emersion on irc.freenode.net.
 
 ## Running
 
-If you're using Sway you can start mako on launch by putting `exec mako` in
-your configuration file.
 
-If you are using elogind, you might need to manually start a dbus user session:
-`dbus-daemon --session --address=unix:path=$XDG_RUNTIME_DIR/bus`
+`mako` will run automatically when a notification is emmited. This happens via
```
Comment (emersion) [inline README.md]: Typo: emitted

File: README.md [hunk 3]
```diff
@@ -12,11 +12,19 @@ Feel free to join the IRC channel: ##emersion on irc.freenode.net.
...
+DBus-activation, so you don't really need to explicitly start it up (this also
```
Comment (emersion) [inline README.md]: "D-Bus activation"

File: README.md (hunk 2 above)
Comment (WhyNotHugo) [inline README.md]:
```suggestion
`mako` will run automatically when a notification is emitted. This happens via
```

File: README.md [hunk 4]
```diff
@@ -12,11 +12,19 @@ Feel free to join the IRC channel: ##emersion on irc.freenode.net.
 
 ## Running
 
-If you're using Sway you can start mako on launch by putting `exec mako` in
-your configuration file.
 
-If you are using elogind, you might need to manually start a dbus user session:
-`dbus-daemon --session --address=unix:path=$XDG_RUNTIME_DIR/bus`
+`mako` will run automatically when a notification is emitted. This happens via
+DBus-activation, so you don't really need to explicitly start it up (this also
```
Comment (WhyNotHugo) [inline README.md]:
```suggestion
D-Bus activation, so you don't really need to explicitly start it up (this also
```

**Pull request merged**